modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -2,38 +2,124 @@
+* ForwardToConsole to forward logs to the system console.
+* ForwardToKsmg to forward logs to the kernel log buffer.
+* ForwardToSyslog to forward to a syslog daemon.
+* ForwardToWall to forward messages to user's wall.
+<3> Specify the maximum time to store journal entries. Enter digits to specify seconds. Or 
+include a unit: "year", "month", "week", "day", "h" or "m". Enter 0 to disable. Default is `1month`. 
+<4> Congifure rate limiting. If, during the time interval defined by `RateLimitIntervalSec`, more logs than specified in `RateLimitBurst` 
+are received, all further messages within the interval are dropped until the interval is over. We recommend setting 
+`RateLimitInterval=1s` and `RateLimitBurst=10000`, which are the defaults.
+<5> Specify how logs are stored. Default is `persistent`: 
+* `volatile` to store logs in memory in `/var/log/journal/`. 
+* `persistent` to store logs to disk  in `/var/log/journal/`. systemd creates the directory if it does not exist. 
+* `auto` to store logs in  in `/var/log/journal/` if the directory exists. If it does not exist, systemd temporarily stores logs in `/run/systemd/journal`.
+* `none` to not store logs. systemd drops all logs.
+<6> Specify the timeout before synchronizing journal files to disk for *ERR*, *WARNING*, *NOTICE*, *INFO*, and *DEBUG* logs. 
+systemd immediately syncs after receiving a *CRIT*, *ALERT*, or *EMERG* log. Default is `1s`.
+<7> Specify the maximum size the journal can use. Defaults to `8g`.
+<8> Specify how much disk space systemd must leave free. Defaults to `20%`.
+<9> Specify the maximum size for individual journal files stored persistently in `/var/log/journal`. Defaults to `10M`.
++
+[NOTE]
+====
+If you removing the rate limit, you might see increased CPU utilization on the
+system logging daemons as it processes any messages that would have previously
+been throttled.
+====
++
+For more information on systemd settings, see link:https://www.freedesktop.org/software/systemd/man/journald.conf.html[https://www.freedesktop.org/software/systemd/man/journald.conf.html]. The default settings listed on that page might not apply to {product-title}.
++
+// Defaults from https://github.com/openshift/openshift-ansible/pull/3753/files#diff-40b7a7231e77d95ca6009dc9bcc0f470R33-R34
+
+. Convert the `journal.conf` file to base64:
++
+----
+$ export jrnl_cnf=$( cat /journald.conf | base64 -w0 )
+----
 
-. Add the following to the *_/etc/systemd/journald.conf_* file:
+. Create a new MachineConfig for master and/or worker and add the `journal.conf` parameters:
++
+For example: 
 +
+[source,yaml]
 ----
-# Disable rate limiting
-RateLimitInterval=1s
-RateLimitBurst=10000
-Storage=volatile
-Compress=no
-MaxRetentionSec=30s
+
+....
+
+config:
+  storage:
+    files:
+    - contents:
+        source: data:text/plain;charset=utf-8;base64,${jrnl_cnf}
+        verification: {}
+      filesystem: root
+      mode: 644
+      path: journald.conf <3>
```

Comment:
The mco won't work, filed bug https://bugzilla.redhat.com/show_bug.cgi?id=1801016.